src/TTrigger.cpp

```diff
@@ -214,21 +241,22 @@ bool TTrigger::setRegexCodeList(QStringList regexList, QList<int> propertyList)
         }
 
         if (propertyList[i] == REGEX_COLOR_PATTERN) {
-            QRegularExpression regex = QRegularExpression(QStringLiteral(R"(FG(\d+)BG(\d+))"));
-            QRegularExpressionMatch match = regex.match(regexList[i]);
-
-            int _pos = match.capturedStart();
+            int textAnsiFg = scmIgnored;
+            int textAnsiBg = scmIgnored;
+            // Decode the pattern string to the colour codes wanted:
+            TTrigger::decodeColorPatternText(regexList.at(i), textAnsiFg, textAnsiBg);
 
-            if (_pos == -1) {
-                mColorPatternList.push_back(nullptr);
+            if (textAnsiBg == scmIgnored && textAnsiFg == scmIgnored) {
+                setError(QStringLiteral("<b><font color='blue'>%1</font></b>")
+                                 .arg(tr("Error: in item %1, no colors to match were set - at least <i>one</i> of the foreground or background must not be <b>ignored</b>.")
```

Comment:
Nested b tags won't produce

Reply:
I believe **bold** is used for other messages in this position in the Editor. I will convert the inner one to *italics* - as it is picking out the control settings' *values* that are a problem.

Reply:
Must not be ignored -- must be given?

Reply:
*Ignored* is the significant word in the reported settings *`"Foreground color ignored"`* cannot be set at the same time as *`"Background color ignored"`* - because then it is not a *Color trigger* (or rather it would fire on EVERYTHING)!